io/common/virtual_bind_unbind.py

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/python
+
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright: 2018 IBM
+# Author: Harsha Thyagaraja <harshkid@linux.vnet.ibm.com>
+# Author: Bismruti Bidhibrata Pattjoshi<bbidhibr@in.ibm.com>
+
+import os
+import time
+import netifaces
+from avocado.utils import genio
+from avocado import Test
+from avocado import main
+from avocado.utils import process
+from avocado.utils.software_manager import SoftwareManager
+from avocado.utils.process import CmdError
+from avocado.utils.network.interfaces import NetworkInterface
+from avocado.utils.network.hosts import LocalHost
+
+
+class VirtualizationDriverBindTest(Test):
+
+    """
+    virtualized devices can be bound and unbound to drivers.
+    This test verifies that for a given virtualized device.
+    :param device: Name of the virtualized device
+    """
+
+    def setUp(self):
+        """
+        Identify the virtualized device.
+        """
+        smm = SoftwareManager()
+        for pkg in ["net-tools"]:
+            if not smm.check_installed(pkg) and not smm.install(pkg):
+                self.cancel("%s package is need to test" % pkg)
+        interfaces = netifaces.interfaces()
+        self.virtual_device = self.params.get('virtual_device')
+        self.virtual_slot = self.params.get('virtual_slot')
+        if "T" in self.virtual_slot:
+            self.virtual_slot = self.virtual_slot.split("-T")[0]
```

Comment:
why do you need this ?  the user will be giving input without -T
virtual_slot: U9009.42A.13C6F1W-V2-C3

so no need to remove it

Reply:
ok

Reply:
@smruti77 I got you, when a user finds slot using lscfg so he will have -T appended..  no issue you can keep for such cases